mesa/agent.py

```diff
@@ -5,20 +5,27 @@
 Core Objects: Agent
 
 """
+# mypy
+from typing import Any
+if False:
+    from .model import Model
 
 
 class Agent:
     """ Base class for a model agent. """
-    def __init__(self, unique_id, model):
+    def __init__(self, unique_id: int, model: Model) -> None:
         """ Create a new agent. """
         self.unique_id = unique_id
         self.model = model
         self.pos = None
 
-    def step(self):
+    def step(self) -> None:
         """ A single step of the agent. """
         pass
 
+    def advance(self) -> None:
+        pass
+
     @property
-    def random(self):
+    def random(self) -> Any:
```

Comment:
I tried setting this method to return `random.Random`, but it said `random.Random` is undefined, so I set this to `Any` instead.